terraform/ssh.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/bash
 
-ssh ubuntu@`terraform output ip`
+ssh ubuntu@`terraform output host_ip`
```

Comment:
Is it useful to have the `./ssh.sh` script in the repo if only one person can use it?